js/build.js

```diff
@@ -173,7 +173,14 @@
   $(document)
     .on('show.bs.collapse', Collapsible.SELECTORS.collapse, function(){
       // Immediately when the expand action is fired
-      Collapsible.prototype.toggleChevron($(this).attr('id'), true);
+      var id = $(this).attr('id');
+      var label = $('[data-toggle="collapse"][data-target="#' + id + '"]').text().trim();
```

Comment:
any reason why this would fail? just checking :)

Reply:
@squallstar Yup. I've checked.

- There's always going to be an ID.
- There's a possiblity of `$('[data-toggle="collapse"][data-target="#' + id + '"]')` returning nothing. If so, `$('[data-toggle="collapse"][data-target="#' + id + '"]').text().trim()` will still work and label will just be an empty string. Not a problem to the subsequent code.